apis/projectcontour/v1alpha1/contourconfig.go

```diff
@@ -76,6 +76,11 @@ type ContourConfigurationSpec struct {
 	// +optional
 	Policy *PolicyConfig `json:"policy,omitempty"`
 
+	// GlobalCircuitBreakerDefaults specifies default circuit breaker budget across all services.
+	// If defined, this will be used as the default for all services.
+	// +optional
+	GlobalCircuitBreakerDefaults *GlobalCircuitBreakerDefaults `json:"circuitBreaker,omitempty"`
```

Comment:
I think it would make sense to move this to under `ClusterParameters` (so `spec.envoy.cluster.circuitBreakers`)